docs/discover/view-document.asciidoc

```diff
@@ -1,25 +1,45 @@
-[[discover-document-context]]
-== View surrounding documents
+[[discover-view-document]]
+== View a document
 
-Once you've narrowed your search to a specific event in *Discover*,
-you can inspect the documents that occurred
-immediately before and after the event.
-To view the surrounding documents, your index pattern must contain time-based events.
+Once you've found a document of interest in Discover there are 2 more ways to view it.
+
+[float]
+[[discover-view-single-document]]
+=== View single document
 
 . In the document table, click the expand icon (>).
-. Click *View surrounding documents.*
+. Click **View single document**
 +
-In the context view, documents are sorted by the time field specified in the index pattern
+This allows you to view the document in two ways. **Table** displays the document's fields row by row.
+**JSON** (JavaScript Object Notation) allows you to take a look at how the document is returned from Elasticsearch.
++
+[role="screenshot"]
+image::images/discover-view-single-document.png[Discover single document view]
+
+The link is valid for the time the document is available in Elasticsearch. If you need another way to view single
```

Comment:
"If you need another way to view a single...."

Could this be?

"If you need to view a single document for a longer time, extend **Table** and ...."

Reply:
This would change the meaning of this, because it by adding another view, the doc wouldn't be available for a longer time. what I'm trying to say is, that a user could implement additional ways to view to doc.